tests/humanize-utils/test_humanize_hex_str.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from eth_utils import humanize_hex_str
+
+
+@pytest.mark.parametrize(
+    "value,expected",
+    (
+        ("000102030405060708090a0b0c0d0e0f101112131415161718191a1b1c1d1e1f", "0001..1e1f"),
+        ("30405060", "3040..5060"),
+        ("30060", "30060"),
```

Comment:
Hex strings also often start with `0x`. I'd like to see a couple examples, including one above and below the display boundary. Presumably the 0x should be stripped before calculation.